src/main/java/org/jabref/model/metadata/MetaData.java

```diff
@@ -31,13 +31,12 @@
 import com.google.common.eventbus.EventBus;
 import com.tobiasdiez.easybind.optional.OptionalBinding;
 import com.tobiasdiez.easybind.optional.OptionalWrapper;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 @AllowedToUseLogic("because it needs access to citation pattern and cleanups")
 public class MetaData {
 
     public static final String META_FLAG = "jabref-meta: ";
+    public static final String META_FLAG_010 = "jabref-meta-0.1.0";
```

Comment:
Include the string `Version` in the constant name:

```suggestion
    public static final String META_FLAG_VERSION_010 = "jabref-meta-0.1.0";
```